Makefile

```diff
@@ -60,15 +63,19 @@ setup-venv: .venv/bin/python
 
 format: setup-venv
 	@clang-format -i \
+		examples/*.c \
+		include/*.h \
 		src/*.c \
 		src/*.h \
 		src/*/*.c \
+		src/*/*.cpp \
 		src/*/*.h \
 		tests/unit/*.c \
 		tests/unit/*.h
 	@.venv/bin/black tests
 .PHONY: format
 
 style: setup-venv
+	@.venv/bin/python ./scripts/run-clang-format.py -r examples include src tests/unit
```

Comment:
_nit_: Can we call this `check-clang-format`?  When reading this for the first time, I got the impression that this would actually format the files :)